source/documentation/getting_started/get_started.erb

```diff
@@ -135,6 +139,89 @@ You can practice deploying an app by deploying a test static HTML page.
 
     If you do not specify the app name in `cf push`, the name specified in the manifest file is used.
 
-    The static HTML page is now available at your [app domain](/orgs_spaces_users.html#regions).
+The static HTML page is now available at your [app domain](/orgs_spaces_users.html#regions).
+
+For a production app, you should read the [production checklist](deploying_apps.html#production-checklist).
+
+## Deploy a test static HTML page using Cloud Foundry API v3
+
+You can practice deploying an app by deploying a test static HTML page.
+
+The following instructions tell you how to deploy a static HTML page using [v3 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v3). This version of the API allows you to push an app with no downtime without needing to use a [blue-green deployment](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html).
```

Comment:
```suggestion
The following instructions tell you how to deploy a static HTML page using [v3 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v3). This version of the API allows you to push an app with no downtime without needing to use a [blue-green deployment](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) plugin.
```
This is just a suggestion, but I think that new users may be confused by what it means otherwise.